package.json

```diff
@@ -3,14 +3,15 @@
   "version": "1.0.2",
```

Comment:
What do you think about bumping this to `2.0.0`?

Reply:
I think that's wise -- adding `"type": "module"` to `package.json` is a breaking change, so a major version bump would keep us semver compliant. I set it to `2.0.0-rc.1` with the idea that we would let it simmer for a bit before we publish a 2.0.0 and update the [latest dist-tag](https://docs.npmjs.com/cli/v10/commands/npm-dist-tag) to the 2.x branch.